src/entities/communitySbt.ts

```diff
@@ -62,6 +62,41 @@ export type GetBadgesStatusArgs = {
     potentialClaimingBadgeTypes: Array<number>;
 }
 
+export type NonProgramaticBadgeResponse = {
+    address: string;
+    badge: string;
+    awardedTimestamp: number;
+    mintedTimestamp: number;
+}
+
+export const NON_SUBGRAPH_BADGES_SEASONS: Record<number, string[]>  = {
```

Comment:
I think this enums belong to FE UI stuff. I would advise we use badgeType enum for the purpose here.

Reply:
refactored in 6d7aae2671da494ef8b743f0d8a24f7c73dbd490